### PR TITLE
Add embed subcommand

### PR DESCRIPTION
**Pull request by mooziii**

### CODE REVIEW

File: src/main/kotlin/net/axay/blaubot/commands/Admin.kt [hunk 1]
```diff
@@ -25,6 +32,16 @@ object Admin : SlashCommand(
         interaction.command.subCommands["chat"]?.options?.get("message")?.string()?.let {
             interaction.channel.createMessage(it)
         }
+        
+        interaction.command.subCommands["embed"]?.options?.get("message")?.string()?.let {
+            interaction.acknowledge(false).followUp {
+                embed {
+                    title = interaction.command.subCommands["embed"]?.options?.get("title")?.string()
+                    image = interaction.member.asUser().avatar.url
+                    description = it
```
Comment (jakobkmar) [inline src/main/kotlin/net/axay/blaubot/commands/Admin.kt]: `description = it` seems to be wrong, as `it` is the embed builder in this case, not the description.

File: src/main/kotlin/net/axay/blaubot/commands/Admin.kt [hunk 2]
```diff
@@ -25,6 +32,16 @@ object Admin : SlashCommand(
         interaction.command.subCommands["chat"]?.options?.get("message")?.string()?.let {
             interaction.channel.createMessage(it)
         }
+        
+        interaction.command.subCommands["embed"]?.options?.get("message")?.string()?.let {
+            interaction.acknowledge(false).followUp {
+                embed {
+                    title = interaction.command.subCommands["embed"]?.options?.get("title")?.string()
+                    image = interaction.member.asUser().avatar.url
```
Comment (jakobkmar) [inline src/main/kotlin/net/axay/blaubot/commands/Admin.kt]: Why do you want to display the avatar of the user who created the embed?

**Review by jakobkmar — CHANGES_REQUESTED** [review]

File: src/main/kotlin/net/axay/blaubot/commands/Admin.kt (hunk 1 above)
Comment (mooziii) [reply src/main/kotlin/net/axay/blaubot/commands/Admin.kt]: No, according to IntelliJ, `it` is the string and` this` is the EmbedBuilder

File: src/main/kotlin/net/axay/blaubot/commands/Admin.kt (hunk 2 above)
Comment (mooziii) [reply src/main/kotlin/net/axay/blaubot/commands/Admin.kt]: I thought you can see who sent the message. I can also replace it with the server icon

File: src/main/kotlin/net/axay/blaubot/commands/Admin.kt (hunk 1 above)
Comment (jakobkmar) [reply src/main/kotlin/net/axay/blaubot/commands/Admin.kt]: Ah yes you are right.
But please only get the embed command options once and save them inside of a variable.
`val embedOptions = interaction.command.subCommands["embed"]?.options`

Comment (jakobkmar) [reply src/main/kotlin/net/axay/blaubot/commands/Admin.kt]: Then you can do `embedOptions?.get("title")` and `embedOptions?.get("message")`

Comment (mooziii) [reply src/main/kotlin/net/axay/blaubot/commands/Admin.kt]: Did it

**Review by jakobkmar — APPROVED** [review]

**Pull request merged**